datafusion/substrait/src/logical_plan/consumer.rs

```diff
@@ -255,6 +257,27 @@ async fn except_rels(
     Ok(rel)
 }
 
+fn from_substrait_names(names: &[String]) -> Result<TableReference> {
```

Comment:
minor: I think it would be good to give this more specific name like `table_reference_from_substrait_names`